pkg/apis/events/v1alpha1/cloudstoragesource_conversion_test.go

```diff
@@ -102,3 +104,50 @@ func TestCloudStorageSourceConversion(t *testing.T) {
 		}
 	}
 }
+
+func TestCloudStorageSourceConversionBetweenV1(t *testing.T) {
```

Comment:
can this be merged with the BetweenV1beta1 test, and you add both versions to the versions list?

Reply:
If we want to combine them, I need to add a `if` to differentiate between v1 and v1beta1 since v1beat1 keeps status.ServiceAccountName and sepc.PayloadFormat. I think that may be hard to read. That's why I separate them in two methods.